doc_admin/AdminManual.adoc

```diff
@@ -728,7 +729,10 @@ When SINGLEUSER is set to YES (default) the program starts with this user and al
 
 If SINGLEUSER is set to NO the following login window appears when the program is started:
 
-image:image24.png[Login.png,width=305,height=148]
+image:image27.png[UsersListLogin.png,width=305,height=148]
```

Comment:
Why not just change the images to the actual size of image24: 373 X 146?

Reply:
I left the same size as the previous image. Where did you find the size 373 X 146?

Reply:
also the previous image had wrong size indeed, you can take it from the [image itself](https://raw.githubusercontent.com/informatici/openhospital-doc/63fe9282b8ee4c30a7c29e5fb5177f75b20a3747/doc_admin/media/image24.png) <-- inspect the html
But what I find is 375x139, not 373x146 @dbmalkovsky

Reply:
@mwithi Yesterday I took a screen shot of the current logon dialog on Linux with JDK 11 and the image parameters are 373 x 146.

Reply:
@dbmalkovsky ok, that could depend on the jdk specific rendering, but I think we should use the actual sizes of the imagefile uploaded on the repository, wdyt?

Reply:
Sounds good to me.